songhyun/app.js

```diff
@@ -42,6 +44,137 @@ app.post("/join", async (req, res) => {
     [name, password, email]
   );
   res.status(201).json({ message: "user created" });
+
+// 글 등록(과제3)
+app.post("/post", async (req, res) => {
+  const { title, content, user_id } = req.body;
+
+  await myDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id
+    ) VALUES (?, ?, ?);
+    `,
+    [title, content, user_id]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+// 글 보기(과제4)
+app.get("/post/view", async (req, res) => {
+  await myDataSource.query(
+    `SELECT
+      posts.user_id AS userId,
+      posts.id AS postingId,
+      posts.imageUrl AS postingImageUrl,
+      posts.content AS postingContent,
+      users.userProfileImage
+    FROM posts
+    INNER JOIN users ON posts.user_id = users.id`,
+    (err, rows) => {
+      res.status(200).json({ data: rows });
+    }
+  );
+});
+
+// 유저가 작성한 글 보기(과제5)
+app.get("/user/view", async (req, res) => {
```

Comment:
```suggestion
app.get("/users", async (req, res) => {
```
```suggestion
app.get("/user-list", async (req, res) => {
```
이 부분도 마찬가지 입니다. 

또한 자원 네이밍은 위와 같이 여러 방식으로 작성할 수 있는데 자세한 내용은 RESTful 세션에서 다룰 예정이니 지금은 이런 느낌이구나~ 정도만 확인 후 넘어가주시면 되겠습니다.